content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1042,6 +1042,19 @@ Used on: Pod
 The Job controller in the kube-controller-manager sets this annotation for Pods
 created with Indexed [completion mode](/docs/concepts/workloads/controllers/job/#completion-mode).
 
+### batch.kubernetes.io/cronjob-scheduled-timestamp
+
+Type: Annotation
+
+Example: `batch.kubernetes.io/cronjob-scheduled-timestamp: "2016-05-19T03:00:00-07:00"`
+
+Used on: Jobs and Pods controlled by CronJobs
+
+This annotation is used to record the original (expected) creation timestamp for a Job,
+when that Job is part of a CronJob.
+The control plane sets the value to that timestamp in RFC3339 format. If the Job belongs to a CronJob
+with a timezone specified, then the timestamp is in that timezone. Otherwise, the timestamp is in controller-manager's local time.
```

Comment:
nit: this text doesn't make it completely clear what happens if you have multiple replicas of k-c-m and, perhaps by accident, they have different local timezones.

Reply:
We may target this in the GA, @soltysh  wdyt?

Reply:
We can _document_ the behavior even for beta. However, leaving the detail out doesn't block a release at beta quality.

Reply:
So based on the already merged implementation we will fallback to UTC, because if the value isn't set we use empty timezone, which so happens resolves to UTC, see https://github.com/kubernetes/kubernetes/blob/ddb201336382808b756a140b7cf2039c6de6f846/pkg/controller/cronjob/utils.go#L219-L227 So I'm fine with Tim's proposal to make it explicit, which is definitely better than my initial approach for leaving it for later.

Reply:
That sounds ideal all round.

Reply:
@helayoty this needs update and it's good to go
```suggestion
with a timezone specified, then the timestamp is in that timezone. Otherwise, the timestamp is in UTC.
```